smarts/env/hiway_env.py

```diff
@@ -62,8 +62,6 @@ class HiWayEnv(gym.Env):
             used to specify envision's data replay output directory
         zoo_workers:
             List of (ip, port) tuples of Zoo Workers, used to instantiate remote social agents
-        auth_key:
-            Authentication key of type string for communication with Zoo Workers
```

Comment:
gRPC authentication provides three [authentication](https://grpc.io/docs/guides/auth/) mechanisms, namely, (i) SSL, (ii) ALTS, and (iii) Token-based authentication with Google. The most suitable option for this project would be SSL.  

Given SSL requires certificates, I thought it would unnecessarily degrade user experience and make it harder for the end user. Hence, I have removed and left out the authentication feature in this pull request.